### DISCUSSION TITLE
How to use regex in import -L queries?

### DISCUSSION THREAD

**Discussion by dustinpaluch** [beetbox/beets#5256] [Q&A] [answered]
If I have an artist with three albums:

Foo
Foo 2
Foo 3

And I want to re-import Foo with `beet import -L <query>`, how do I query for Foo and not Foo 2 and 3? `beet import -L Foo$` doesn't work.

**Comment by arogl** [comment, marked as answer]
> If I have an artist with three albums:
> 
> Foo Foo 2 Foo 3
> 
> And I want to re-import Foo with `beet import -L <query>`, how do I query for Foo and not Foo 2 and 3? `beet import -L Foo$` doesn't work.

try `beet imp 'albumartist::^Foo$'`

If on windows using `cmd.exe` the swap the `'` for `"`

**Reply by dustinpaluch** [reply]
This works if I include `-L` and change `albumartist` to `album`. Thanks.

**Reply by arogl** [reply]
What you start replying before the ☕ kicks in, at least a pointer in the right direction
